Formula/s/swiftlint.rb

```diff
@@ -17,7 +17,7 @@ class Swiftlint < Formula
     sha256                               x86_64_linux:   "fbbc56fccfcfcd34564feb7325567e2ff3638d3c609396a5c4aa13311c7b26e0"
   end
 
-  depends_on xcode: ["14.0", :build]
+  depends_on xcode: ["15.3", :build]
```

Comment:
This won't work as Xcode 15.3 isn't released yet. It is still in beta.

Sonoma CI runners are currently on Xcode 15.1 thus this results in skipping all tests and giving a fake pass.

Reply:
Good hint. I misinterpreted the last comments in https://github.com/Homebrew/homebrew-core/pull/153931. It sounded like something got fixed/updated already.